atef/widgets/config/run_active.py

```diff
@@ -98,3 +101,80 @@ def __init__(self, *args, data: PreparedDescriptionStep, **kwargs):
     def _setup_ui(self) -> None:
         self.title_label.setText(self.bridge.origin.get().name)
         self.desc_label.setText(self.bridge.origin.get().description)
+
+
+class SetValueRunWidget(DesignerDisplay, DataWidget):
```

Comment:
I like all the elements here especially the intuitive options menu, but this widget renders confusingly at large sizes

![image](https://user-images.githubusercontent.com/10647860/232605797-abdfa96e-e449-4b85-ba01-9d696aae57f4.png)

![image](https://user-images.githubusercontent.com/10647860/232605852-48017c85-90e2-4543-ad39-3aedbc85d8b3.png)

I don't have any immediate suggestions about the layout, but I think the reason the large one feels bad is because the two most important bits of information (target, value) end up far apart instead of close together.

If you've had enough of fighting with layouts I can spend some time trying to muck with this too, it's always a maze of size policies...

Reply:
Maybe a small actionable idea for this is to enforce a maximum arrow length and put a horizontal spacer immediately left of the x, since we don't actually want all the values to run away from their targets, do we?

Reply:
Turns out the a few widgets needed their width policies set to maximum, alongside the horizontal spacer to the right of the delete button 👍

Reply:
addressed in [93849d0](https://github.com/pcdshub/atef/pull/159/commits/93849d0eee9f6b3d14c1a77d6be658f0e66cce93)